src/config.js

```diff
@@ -66,7 +66,11 @@ export function applyConfigToArgv({
     const expectedType = options[decamelizedOptName].type ===
       'count' ? 'number' : options[decamelizedOptName].type;
 
-    const optionType = typeof configObject[option];
+    const optionType = (
+      Array.isArray(configObject[option]) ?
```

Comment:
While you're in here, could you also add support for number types [like this one](https://github.com/mozilla/web-ext/blob/cdb5b003afceea9426490fc21897cbd6e8fc9a3d/src/program.js#L393)?

Reply:
Nevermind, sorry, I thought that it would need special handling but it does not.